packages/libspiro.rb

```diff
@@ -0,0 +1,21 @@
+require 'buildsystems/autotools'
+
+class Libspiro < Autotools
+  description 'LibSpiro is an adaptation of Spiro formula and functions into a sharable library'
+  homepage 'https://github.com/fontforge/libspiro'
+  version '20221101'
+  license 'GPL3+'
+  compatibility 'all'
+  source_url 'https://github.com/fontforge/libspiro.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
```

Comment:
Missing binary hash?

Reply:
I have no idea how that even happened, given that `crew upload` should replace both `aarch64` and `armv7l` given that they have the same base hash.

Reply:
The process probably shouldn't rely on a template but rather generate the entire section if it isn't present.